Sources/Bow/Typeclasses/Monoidal.swift

```diff
@@ -0,0 +1,23 @@
+/// The Monoidal type class adds an identity element to Semigroupal type class by defining the function identity
+///
+/// identity returns a specific identity `Kind<F, A>` value for a given type F and A.
+///
+/// This type class complies with the following law:
+/// fa.product(identity) == identity.product(fa) == identity
+///
+/// In addition, the laws of Semigroupal type class also apply.
+///
+/// Currently, Monoidal instance is defined for Option
```

Comment:
```suggestion
/// - Note:
/// Currently, Monoidal instance is defined for Option
```

It will be rendered as a note, you can see the result [here](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/Note.html#//apple_ref/doc/uid/TP40016497-CH39-SW1)

Reply:
Oh neat, TIL